### DISCUSSION TITLE
Mark as Not Suspicious

### DISCUSSION THREAD

**Discussion by nnnnicholas** [anonaddy/anonaddy#668] [Feature Requests]
Some sites I deal with send emails that are always marked as suspicious. Something wrong with their setup, no doubt.

I would like a button to "Mark as Not Suspicious" to remove this warning from future emails from this sender to this alias. 

<img width="730" alt="image" src="https://github.com/user-attachments/assets/a516ab97-c3db-4e44-98f0-8e913e11f927">

**Comment by batmanvane** [comment]
Is there a way to turn off this warning. my use case is the following:

customer sends request via wordpress form from a trusted server, where via shortlink his email is put into "sent from" and the forms content is forwarded to a recipient email, 

this is done to be able to reply to the right adress by simply hitting "reply" in an arbitrary email programm/webmailer . of course the email domain from the customers is in most cases not tied to the sending server and thus looks like a spoofing attempt.

hope i explained ok. it is a kind of light ticket system.

or can i somehow configure the DNS entries from the sending server in a right way?

**Reply by willbrowningme** [reply]
You can simply set a `Reply-To` header on those wordpress form emails. Then they will not fail DMARC checks.

**Reply by batmanvane** [reply]
Thanks for the suggestion. I already tried - didn't work.

**Reply by batmanvane** [reply]
I fixed the problem. My SMTP was setup with an incorrect domain.
